rsocket/statemachine/RSocketStateMachine.cpp

```diff
@@ -76,25 +76,27 @@ void RSocketStateMachine::setResumable(bool resumable) {
   remoteResumeable_ = isResumable_ = resumable;
 }
 
-bool RSocketStateMachine::connectServer(
+void RSocketStateMachine::connectServer(
     yarpl::Reference<FrameTransport> frameTransport,
     const SetupParameters& setupParams) {
   setResumable(setupParams.resumable);
-  return connect(std::move(frameTransport), true, setupParams.protocolVersion);
+  CHECK(connect(std::move(frameTransport), setupParams.protocolVersion));
+  sendingPendingFrames();
 }
 
+//TODO: ensure the return value is not ignored!
```

Comment:
no, I was going to address this in the following diff. I didn't know about [[nodiscard]]. thanks! ;)